test/views/helper_methods_test.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+require 'test_helper'
+
+module ActionView
+  include Devise::Views::Helpers
+end
+
+class ViewsHelperMethodsTest < Devise::IntegrationTest
+
+  test 'Action View includes Devise::Views::Helpers' do
+    assert_includes  ActionView.ancestors, Devise::Views::Helpers
```

Comment:
```suggestion
    assert_includes ActionView.ancestors, Devise::Views::Helpers
```

Reply:
Can we get this pulled?

Reply:
Thank you for the review. Could we get this pull request submitted?